bazel/utils/exec_test.bzl

```diff
@@ -5,7 +5,7 @@ def _exec_test(ctx):
     # the executable returned by another rule.
     template = """#!/bin/bash
 ARGS={argv}; ARGS+=("$@")
-exec {script} "$ARGS[@]"
+exec {script} "${{ARGS[@]}}"
```

Comment:
Why are double braces required here?  I'm not a bash array expansion expert.  The examples I see online are only using a single set of braces.

Reply:
this is a string expanded with python format. `{`, for python format, needs to be escaped as `{{`. bash sees a single `{'.

Reply:
ah